### PR TITLE
[microTVM] make RVM memory and number of cores variable

### PR DESCRIPTION
**Pull request by mehrdadh**
With this change we can set the memory size and number of cores for RVM with `vagrant up` command. I kept default values for them so that we don't require to change the tutorials.

### CODE REVIEW

**Comment by mehrdadh** [comment]
cc @areusch

File: apps/microtvm/reference-vm/zephyr/Vagrantfile [hunk 1]
```diff
@@ -18,6 +18,18 @@
 Vagrant.configure("2") do |config|
   config.vm.box = "tlcpack/microtvm-zephyr-2.5"
 
+  if ENV.has_key?("RVM_NUM_CORES")
```
Comment (areusch) [inline apps/microtvm/reference-vm/zephyr/Vagrantfile:21]: maybe we want to prefix all env vars with TVM_?

Comment (mehrdadh) [reply apps/microtvm/reference-vm/zephyr/Vagrantfile]: Done.

**Review by areusch — APPROVED** [review]
thanks @mehrdadh !

**Pull request merged**